nsis/gvim.nsi

```diff
@@ -74,6 +74,7 @@ Unicode true
 !include "nsDialogs.nsh"
 !include "Sections.nsh"
 !include "x64.nsh"
+!include "StrFunc.nsh"
```

Comment:
what is this for please?

Reply:
Including this plugin was necessary for this code:
```
  StrCmp $R9 "zh-cn" 0 +3
  ${StrRep} $lng_usr $R9 "-" "_"
  Goto Done
  StrCmp $R9 "zh-tw" 0 +3
  ${StrRep} $lng_usr $R9 "-" "_"
```

Reply:
yes fine